python/lsst/synpipe/matchFakes.py

```diff
@@ -279,10 +279,17 @@ def getFakeSources(butler, dataId, tol=1.0,
 
     if ('pixelScale' in extraCols) or ('thetaNorth' in extraCols):
         wcs = cal.getWcs()
-        availExtras['pixelScale']['value'] = wcs.pixelScale().asArcseconds()
+        availExtras['pixelScale']['value'] = wcs.getPixelScale().asArcseconds()
+        xMid = cal.getWidth() // 2
```

Comment:
Please add a comment block here describing why all the below is happening, it may not be obvious to future readers